src/sdk/js/6/controllers/auth/check-token/index.md

```diff
@@ -7,19 +7,21 @@ description: Checks a JWT's validity.
 
 # checkToken
 
-Checks a token validity.
+Checks a token validity.  
+
+If not token is passed in parameter, the SDK will check his internal authentication token.
 
 <br/>
 
 ```js
-checkToken(token);
+checkToken([token]);
 ```
 
 <br/>
 
 | Property | Type              | Description |
 | -------- | ----------------- | ----------- |
-| `token`  | <pre>string</pre> | JWT token   |
+| `token`  | <pre>string</pre> | Optional JWT token   |
```

Comment:
```suggestion
| `token`  | `string` | Optional JWT token   |
```

Reply:
In our current template, we need both the argument name and its type to be in fixed case, but we decided that the grey background was too heavy to the eyes when applied to both columns. 
Hence we decided that the name should be escaped with backticks, and the type should simply put in fixed case without decoration.

Reply:
Ok, I see the problem.
In some cases, the `Type` column contains links (e.g. the Request Object), that must be expressed in Markdown logic, so that the parser transforms them in `router-link`s. The `<pre>` creates a HTML block that prevents the MD parser to take it into account.

We need to find a better solution for styling inline code in different ways...

Reply:
That woud be nice to have some kind of tag dedicated to print something using a fixed font, without all the decoration used with backticked text

Reply:
Problem is... No Markdown will be allowed inside. Thus, no `router-link`s...

Reply:
This is outside the scope of this PR